### PR TITLE
Docblock tweaks

### PR DESCRIPTION
**Pull request by davidyell**
Just a few minor tweaks

### CODE REVIEW

**Comment by davidyell** [comment]
Thanks @dereuromark 👍

File: src/Http/Client.php [hunk 1]
```diff
@@ -442,7 +442,7 @@ protected function _sendRequest(Request $request, $options)
      * @param string $url Either a full URL or just the path.
      * @param string|array $query The query data for the URL.
      * @param array $options The config options stored with Client::config()
-     * @return string A complete url with scheme, port, host, path.
+     * @return string A complete url with scheme, port, host, and path.
```
Comment (ADmad) [inline src/Http/Client.php]: @davidyell Does the Queen's English use comma before "and"? :)

Comment (davidyell) [reply src/Http/Client.php]: https://en.oxforddictionaries.com/explore/what-is-the-oxford-comma

Comment (ADmad) [reply src/Http/Client.php]: Thanks for the reference.

**Pull request merged**